addons/test/misra/misra-test.c

```diff
@@ -160,13 +160,13 @@ static void foo(void)
       {
         for(i = 0; i < 10; i++)
         {
-          if(misra_5_2_func3()) //14.4
+          if(misra_5_2_func3())
```

Comment:
the idea is to report 17.3 here instead. will you report 17.3 in a follow up PR?